libtelemetry/src/main/java/com/mapbox/services/android/telemetry/MapboxTelemetry.java

```diff
@@ -247,9 +258,9 @@ private boolean isNetworkConnected() {
     }
   }
 
-  private void sendEvents(List<Event> events, Callback httpCallback) {
+  private void sendEvents(List<Event> events) {
     if (checkRequiredParameters(accessToken, userAgent)) {
-      telemetryClient.sendEvents(events, httpCallback);
+      telemetryClient.sendEvents(events, telemetryListeners);
```

Comment:
This should be tackled differently 👀 https://github.com/mapbox/mapbox-events-android/pull/38#discussion_r161520797